### DISCUSSION TITLE
Updates not published to deployed domain

### DISCUSSION THREAD

**Discussion by rishabhchaudhary0210** [vercel/community#3523] [Help]
### Summary

I have made a custom domain for my project in vercel.
But whenever i push a commit to git and vercel deploys the updates that particular domain does no get updated.

This domain is not updated
https://the-daily-journal-blog.vercel.app/

This one is updated with commit
https://the-daily-journal-git-master-rishabhchaudhary0210.vercel.app/

Please help as I am unable to figure out what could be the possible problem

### Steps to Reproduce

It can be checked  easily that one link is updated but the other one is not

This domain is not updated
https://the-daily-journal-blog.vercel.app/

This one is updated with commit
https://the-daily-journal-git-master-rishabhchaudhary0210.vercel.app/

**Comment by amyegan** [comment]
Hey @rishabhchaudhary0210. It looks like https://the-daily-journal-blog.vercel.app redirects to https://the-daily-journal-git-master-rishabhchaudhary0210.vercel.app. If you want to use https://the-daily-journal-blog.vercel.app as the production domain, you can change the domain configuration to accomplish that. 

The https://the-daily-journal-blog.vercel.app domain would need to have `Redirect to` changed to `No Redirect` with `Git Branch` left blank. You can remove the https://the-daily-journal-git-master-rishabhchaudhary0210.vercel.app domain if you no longer want to use it as your production domain.

**Reply by rishabhchaudhary0210** [reply]
Hi @amyegan  I know it is redirected currently but i did it only due to the reason that I needed to show the updates on the production 
 domain https://the-daily-journal-blog.vercel.app/ 
Due to time constraint the redirection was the temporary solution but it is still not fixed as you can check i have removed the redirection for now. 
The updates are being published on this domain 
https://the-daily-journal-git-master-rishabhchaudhary0210.vercel.app/

But this one still shows the previous version of the project
https://the-daily-journal-blog.vercel.app/

Also I tried removing this domain and again adding it to same project it still showed the previous version and if i assign a new domain that works well So I am really unable to figure out what could be the cause of this issue.

**Reply by amyegan** [reply]
It looks like you got it working. I see https://the-daily-journal-git-master-rishabhchaudhary0210.vercel.app/ and https://the-daily-journal-blog.vercel.app/ both pointing at the same deployment now.
